extensions/amp-base-carousel/0.1/carousel.css

```diff
@@ -57,17 +57,17 @@
  * calculating the 100% height (or width) of the slide.
 */
 /* Firefox */
-.i-amphtml-carousel-scroll[hide-scrollbar="true"] {
-  scrollbar-width: none; 
+.i-amphtml-carousel-scroll[hide-scrollbar='true'] {
```

Comment:
This change appears unrelated, was this from prettier formatting?

Reply:
Yea this whole file was formatted in [11b3b3a](https://github.com/ampproject/amphtml/pull/33279/commits/11b3b3a3dafe243baed98d240b1514813fde64e6)